### PR TITLE
1682 molecularize orangelight

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: roles/pulibrary.orangelight/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Role Name
+=========
+
+Installs Princeton's version of Blacklight
```
Comment (hackmastera) [inline roles/pulibrary.orangelight/README.md:4]: This description needs to be updated

File: roles/pulibrary.orangelight/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+License
+-------
+
+BSD
+
+Author Information
+------------------
+
+An optional section for the role authors to include contact information, or a
+website (HTML is not allowed).
```
Comment (hackmastera) [inline roles/pulibrary.orangelight/README.md:36]: Delete this if we aren't using it?

File: roles/pulibrary.orangelight/molecule/default/playbook.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  vars:
+    - running_on_server: false
+    - passenger_listen_port: '8080'
+    - rails_app_directory: 'approvals'
+    - rails_app_name: 'approvals'
+    - install_mailcatcher: true
+  roles:
+    - role: pulibrary.postgresql
+    # Specify each pulibrary role here
+    - role: pulibrary.approvals
```
Comment (hackmastera) [inline roles/pulibrary.orangelight/molecule/default/playbook.yml:13]: Should the orangelight role be included?

Comment (christinach) [reply roles/pulibrary.orangelight/molecule/default/playbook.yml:13]: same question for the orangelight role.

**Pull request closed without merging**